.github/workflows/format.yml

```diff
@@ -3,7 +3,7 @@ name: Format
 on:
   push:
     branches:
-      - main
+      - '*'
```

Comment:
Oh this is how you do it?  How come `pull_request` doesn't require that?

Reply:
Interesting. Let's see what happens if I remove the `*`

Reply:
Seems to work the same